### PR TITLE
chezmoi: add traditional Chinese translation

### PR DESCRIPTION
**Pull request by kypkk**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thank for your contribution!

File: pages.zh_TW/common/chezmoi.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> 一個用 Go 語言寫的 dotfile 管理工具
+> 更多資訊: <https://chezmoi.io>.
+
+- 初始化 chezmoi:
+
+`chezmoi init`
+
+- 叫 chezmoi 管理一個 dotfile:
+
+`chezmoi add {{path/to/file}}`
```
Comment (kbdharun) [inline pages.zh_TW/common/chezmoi.md:12]:
```suggestion
`chezmoi add {{檔案/完整/路徑}}`
```
[Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.zh_TW/common/chezmoi.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- 編輯一個已管理的 dotfile :
+
+`chezmoi edit {{path/to/file}}`
```
Comment (kbdharun) [inline pages.zh_TW/common/chezmoi.md:16]:
```suggestion
`chezmoi edit {檔案/完整/路徑}}`
```
[Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

**Review by kbdharun — COMMENTED** [review]
LGTM, Thanks for your contribution, I have some minor suggestions for this page.

We recommend translating the placeholders and paths whenever possible.

File: pages.zh_TW/common/chezmoi.md (hunk 1 above)
Comment (kypkk) [reply pages.zh_TW/common/chezmoi.md:12]: sure

**Review by kbdharun — APPROVED** [review]

File: pages.zh_TW/common/chezmoi.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Chezmoi
+
+> 一個用 Go 語言寫的 dotfile 管理工具
+> 更多資訊: <https://chezmoi.io>.
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:4]: We prefer full-width punctuation on Chinese pages.
```suggestion
> 一個用 Go 語言寫的 dotfile 管理工具。
> 更多資訊：<https://chezmoi.io>.
```

File: pages.zh_TW/common/chezmoi.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- 初始化 chezmoi:
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:6]:
```suggestion
- 初始化 chezmoi：
```

File: pages.zh_TW/common/chezmoi.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi init`
+
+- 叫 chezmoi 管理一個 dotfile:
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:10]:
```suggestion
- 叫 chezmoi 管理一個 dotfile：
```

File: pages.zh_TW/common/chezmoi.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi add {{檔案/完整/路徑}}`
+
+- 編輯一個已管理的 dotfile :
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:14]:
```suggestion
- 編輯一個已管理的 dotfile：
```

File: pages.zh_TW/common/chezmoi.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi edit {檔案/完整/路徑}}`
+
+- 檢視 chezmoi 所做的更動 :
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:18]:
```suggestion
- 檢視 chezmoi 所做的更動：
```

File: pages.zh_TW/common/chezmoi.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi diff`
+
+- 套用所做的更動:
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:22]:
```suggestion
- 套用所做的更動：
```

File: pages.zh_TW/common/chezmoi.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi -v apply`
+
+- 用一個已存在的 git repository 來初始化 chezmoi:
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:26]:
```suggestion
- 用一個已存在的 git repository 來初始化 chezmoi：
```

File: pages.zh_TW/common/chezmoi.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi init {{https://example.com/path/to/repository.git}}`
+
+- 從遠端的 repository 獲取所做的更動:
```
Comment (blueskyson) [inline pages.zh_TW/common/chezmoi.md:30]:
```suggestion
- 從遠端的 repository 獲取所做的更動：
```

**Pull request merged**